setup/make_xdaq_softlinks.sh

```diff
@@ -1,26 +1,23 @@
-mkdir -p ${XDAQ_DOCUMENT_ROOT}/gemdaq/gembase
-mkdir -p ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemutils
-mkdir -p ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemhardware
-mkdir -p ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemreadout
-mkdir -p ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemsupervisor
-mkdir -p ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemcalibration
-mkdir -p ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemHwMonitor
-mkdir -p ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemdaqmonitor
 
-unlink ${XDAQ_DOCUMENT_ROOT}/gemdaq/gembase/html
-unlink ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemutils/html
-unlink ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemhardware/html
-unlink ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemreadout/html
-unlink ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemsupervisor/html
-unlink ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemcalibration/html
-unlink ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemHwMonitor/html
-unlink ${XDAQ_DOCUMENT_ROOT}/gemdaq/gemdaqmonitor/html
+packages=(
```

Comment:
Such refactoring should probably happen on the main branch

Reply:
It will get merged into the main branch when `feature/gemonlindb` is merged in

Reply:
My concern is that it makes the purpose of this branch less clear. But that's only a minor issue :)